packages/components/TestimonialSection.tsx

```diff
@@ -29,11 +37,71 @@ export default function TestimonialSection() {
         my="2rem"
         justifyItems="center"
       >
-        <div>Place Holder</div>
-        <div>Place Holder</div>
-        <div>Place Holder</div>
-        <div>Place Holder</div>
+        {testimonialsList.map((testimonialData) => (
+          <TestimonialCard
+            key={testimonialData.name}
+            name={testimonialData.name}
+            testimonial={testimonialData.testimonial}
+            img={testimonialData.img}
+            borderColor={testimonialData.borderColor}
+          />
+        ))}
       </Grid>
     </Section>
   );
 }
+
+interface TestimonialCardProps {
+  name: string;
+  testimonial: string;
+  img: string;
+  borderColor: string;
+}
+
+function TestimonialCard({
+  name,
+  testimonial,
+  img,
+  borderColor,
+}: TestimonialCardProps) {
+  return (
+    <Box maxW="400px" maxH="400" rounded="lg" shadow="lg" p={2}>
+      <Flex justifyContent="space-between" mb="1">
+        <Heading alignSelf="center" size="sm">
+          {name}
+        </Heading>
+        <Image
+          src={img}
+          alt={`img${name}`}
+          maxW={{ base: '60px', sm: '70px' }}
+          fit="cover"
+          rounded="full"
+          borderStyle="solid"
+          borderWidth="3px"
+          borderColor={borderColor}
+        />
+      </Flex>
+      <Text
+        maxH="300px"
+        overflow="auto"
+        css={{
+          '&::-webkit-scrollbar': {
+            width: '0px',
+          },
+        }}
+      >
+        {testimonial}
+      </Text>
+    </Box>
+  );
+}
+
+const testimonialsList = [
+  {
+    name: 'Name SecondName',
+    testimonial:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempora dolores aspernatur ipsa quisquam! Quod temporibus voluptatem delectus autem ad perferendis iste earum alias molestiae corrupti architecto, vel consectetur error totam sint rem consequatur aut facere? Quidem vitae fuga ab fugit? ',
+    img: 'https://avatars.githubusercontent.com/u/86134825?v=4',
+    borderColor: '#17A9BC',
+  },
+];
```

Comment:
would it be interesting to mount this as a json? thinking about a backend in the future? ps: sorry for the english, i have to use a translator hahaha